### PR TITLE
Refreshed logos - cleanup

### PR DESCRIPTION
**Pull request by erogluorhan**
## Overview
- Remove blue logos (we decided to use only horizontal and vertical layout logos with white and black "xarray" texts), 
- Rename dark and light logos ("h_dark" --> uxarray_logo-h-dark etc.), 
- Remove the older temporary logo (it's now archived in the drive)
- Use website color scheme dependent logo themes, i.e. if dark website use white text logo, or vice versa

## PR Checklist

**General**
- [x] Add appropriate labels
- [x] Filled out the Overview section

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,8 @@
-
-<img src="https://github.com/UXARRAY/uxarray/blob/c8fb393066acd864ef5dceb1956379775a4f21dd/docs/_static/images/logos/h_blue.svg" data-canonical-src="https://github.com/UXARRAY/uxarray/blob/c8fb393066acd864ef5dceb1956379775a4f21dd/docs/_static/images/logos/h_blue.svg" width="450"/><br>
+<picture>
+  <source media="(prefers-color-scheme: dark)" srcset="/docs/_static/images/logos/uxarray_logo-h-light.svg">
+  <source media="(prefers-color-scheme: light)" srcset="/docs/_static/images/logos/uxarray_logo-h-dark.svg">
+  <img alt="Shows a black logo in light color mode and a white one in dark color mode." src="https://user-images.githubusercontent.com/25423296/163456779-a8556205-d0a5-45e2-ac17-42d089e3c3f8.png">
+</picture>
```
Comment (philipc2) [inline README.md:5]: Can we set the `width` to 450 to match with what we had before?

File: docs/conf.py [hunk 2]
```diff
@@ -180,18 +180,18 @@ def __getattr__(cls, name):
...
 
 # # Assuming your `conf.py` has a sibling folder called `_static` with these files
 # html_theme_options = {
 #    "logo": {
-#       "image_light": "_static/images/logos/h_dark.svg",
-#       "image_dark": "_static/images/logos/h_light.svg",
+#       "image_light": "_static/images/logos/uxarray_logo-h-dark.svg",
+#       "image_dark": "_static/images/logos/uxarray_logo-h-light.svg",
 #    }
 # }
```
Comment (philipc2) [inline docs/conf.py:196]: Should this be uncommented?

**Review by philipc2 — CHANGES_REQUESTED** [review]
In the logo rename, there's a mixture of `_` and `-` in the name.

`uxarray_logo-h-dark.svg`

I'd suggested one of the following options, with no preference. 

`uxarray_logo_h_dark.svg`

or 

`uxarray-logo-h-dark.svg`

File: README.md [hunk 3]
```diff
@@ -1,11 +1,11 @@
...
-<img src="https://github.com/UXARRAY/uxarray/blob/c8fb393066acd864ef5dceb1956379775a4f21dd/docs/_static/images/logos/h_blue.svg" data-canonical-src="https://github.com/UXARRAY/uxarray/blob/c8fb393066acd864ef5dceb1956379775a4f21dd/docs/_static/images/logos/h_blue.svg" width="450"/><br>
+<picture>
+  <source media="(prefers-color-scheme: dark)" srcset="/docs/_static/images/logos/uxarray_logo_h_light.svg">
+  <source media="(prefers-color-scheme: light)" srcset="/docs/_static/images/logos/uxarray_logo_h_dark.svg">
+  <img alt="Shows a black logo in light color mode and a white one in dark color mode." src="https://user-images.githubusercontent.com/25423296/163456779-a8556205-d0a5-45e2-ac17-42d089e3c3f8.png">
+</picture>
 
 -----------------
 
-[comment]: <> (<img src='https://raw.githubusercontent.com/UXARRAY/uxarray/a2d893cf597dd1a6e775f9dad029c662c64a39c7/docs/_static/images/logos/uxarray_logo_quad_tri.svg' width='350'>)
```
Comment (erogluorhan) [inline README.md:6]: @philipc2 do you recall what this line originally was for?

File: docs/conf.py (hunk 2 above)
Comment (erogluorhan) [reply docs/conf.py]: I think it should; have done it now

File: README.md (hunk 3 above)
Comment (philipc2) [reply README.md:6]: I think this may be a left over attempt that I commented out. We can remove it

**Comment by erogluorhan** [comment]
> In the logo rename, there's a mixture of `_` and `-` in the name.
> 
> `uxarray_logo-h-dark.svg`
> 
> I'd suggested one of the following options, with no preference.
> 
> `uxarray_logo_h_dark.svg`
> 
> or
> 
> `uxarray-logo-h-dark.svg`

You're right; done it now with all underscores.

File: README.md (hunk 3 above)
Comment (erogluorhan) [reply README.md:6]: Yeah, removed it already

File: README.md (hunk 1 above)
Comment (erogluorhan) [reply README.md]: Done

**Review by aaronzedwick — APPROVED** [review]

**Review by philipc2 — APPROVED** [review]
These changes are great! Tested out the Light/Dark switching on both the README and ReadTheDocs and both are working as expected. Thanks for cleaning this up!

**Pull request merged**